core/services/wifi/WifiManager.py

```diff
@@ -166,6 +166,9 @@ async def connect_to_network(self, network_id: int) -> None:
             network_id {int} -- Network ID provided by WPA Supplicant
         """
         try:
+            saved_networks = await self.get_saved_wifi_network()
+            for network in saved_networks:
+                await self.wpa.send_command_disable_network(network.networkid)
```

Comment:
I'm not sure if if this is the correct approach, disabling every single network when connecting to something else will result in a non connection when restarting the ROV.

Imagine the following scenario: 
Connect the ROV in House1 Wifi
Go to House2 and connect the ROV (House1 wifi not available)
Go back to House1 and there'll be no ethernet connection, since the network is disabled by default. 
We should force connection and not disabling everything else.

Reply:
Got it and agree with you that there is this case. I don't know about a method for forcing the connection without disabling the others, though. Do you know how to do it?

Reply:
From what I could understand from WPA supplicant and WPA cli, "SELECT_NETWORK" does this (disables all networks and connect to the desired one), but it enables them again after, so it forces the connection but keep the enabled ones...enabled.

I will try to see if it indeed works like this. If not, we can add a start logic stating that if all networks are disabled and one of them is available nearby, we enable it.

Reply:
No, "SELECT_NETWORK" indeed disables all the others but the desired, but doesn't enable them back. I've tried doing this (enabling them back after connection) and apparently this approach works (it doesn't connect again to the later enabled, even if it's stronger.

Will add this logic to the method.